src/index.ts

```diff
@@ -83,3 +56,18 @@ const checkMessage = async (message: Message, guildId: string) => {
 
 	return status;
 };
+
+const processCommand = async (msg: Discord.Message) => {
+	if (msg.content.startsWith(process.env.CMD_PREFIX!)) {
+		const args: string[] = msg.content.slice(process.env.CMD_PREFIX!.length, msg.content.length).trim().split(/ +/g);
+		const command = args.shift()!.toLowerCase();
+
+		try {
+			delete require.cache[require.resolve(`./commands/${command}.ts`)];
+			const commandFile = await import(`./commands/${command}.ts`);
```

Comment:
Need to discuss how we handle loading commands - I prefer registering them by hand for a little more compile time type safety.